File: my java coding/arraydynamically.c
#include<stdio.h>
int main()
{
  int i,n,arr[20];
  printf("Enter the number of elements in a array\n");
  scanf("%d",&n);
  for(i=0;i<n;i++)
  {
      printf("\n arr[%d]=",i);
      scanf("%d,arr[i]");
      }
      for(i=0;i<n;i++){
      printf("%d",arr[i]);

      }
      return 0;

}

File: my java coding/arrayDynamically.java
import java.util.Scanner;
class arrayDynamically
{
	public static void main(String args[])
{
	int i=0;
	Scanner kb=new Scanner(System.in);
	System.out.println("enter the array size");
	int s=kb.nextInt();
	int arr[] =new int[s];
	System.out.println("----------------");
	System.out.println("enter the array elements");
	System.out.println("--------------------");
	while(i<s)
{
	arr[i]=kb.nextInt();
	i++;
}
       
	while(s>=0){
            System.out.print(arr[i]+ " ");
            s--;
}
}


}
